pkgs/uf2conv.nix

```diff
@@ -0,0 +1,23 @@
+# Copyright lowRISC contributors.
+#
+# SPDX-License-Identifier: MIT
+{
+  python3Packages,
+  fetchFromGitHub,
+  lib,
+}:
+python3Packages.buildPythonPackage rec {
+  name = "uf2conv";
```

Comment:
This should be `pname` actually. Nix run can then guess the binary name correctly too.